ls8/cpu.c

```diff
@@ -65,13 +65,16 @@ void cpu_load(struct cpu *cpu, char * filepath)
 
     unsigned char val = strtoul(buf, &endptr, 2);
 
+    printf("line: %u, loading ram: %04u\n", address, val);
+
     if (buf == endptr)
     {
       continue;
     }
     // putchar(c);
     
-    cpu_ram_write(cpu, address, val);
+    // cpu_ram_write(cpu, address, val);
+    cpu->ram[address] = val;
```

Comment:
nice. nitpicking, but 
`cpu->ram[address++] = val;`
is also possible. it (should) run your operation first and _then_ increment address (`whereas cpu->ram[++address] = val;` would increment if _before_ running it)